docs/usage/2-projectswift.mdx

```diff
@@ -300,7 +300,7 @@ It represents a list of source files that are part of a target:
     {
       name: 'Globs',
       description:
-        'Path to the source files. They represented by a glob pattern and the compiler flags.',
+        'Path to the source files. They are represented by a [glob pattern](https://facelessuser.github.io/wcmatch/glob/) and the compiler flags.',
```

Comment:
😬 You'll have to extend the component to support descriptions as markdown content. Otherwise it'll show up as you are passing it.

Reply:
You need to update [this line](https://github.com/tuist/tuist/blob/master/docs/components/properties-table.jsx#L35) with something along the lines of:

```js
import { MDXRenderer } from "gatsby-plugin-mdx";

<Table.Cell><MDXRenderer>{prop.description}</MDXRenderer></Table.Cell>
```